packages/core/src/plugins/load.ts

```diff
@@ -0,0 +1,66 @@
+import type {
+  NamedLoksePlugin,
+  PluginFactory,
+  GeneralPluginOptions,
+} from "./create";
+import { PluginsRunner } from "./runner";
+
+function interopRequire(path: string) {
+  const obj = require(path);
+  return obj && obj.__esModule ? obj.default : obj;
+}
+
+function loadPlugin(
+  plugin: PluginName | PluginDefinition,
+  options: GeneralPluginOptions
+): NamedLoksePlugin | null {
+  const pluginName = typeof plugin === "string" ? plugin : plugin.name;
+  const pluginOptions =
+    typeof plugin === "string" ? options : { ...options, ...plugin.options };
+
+  try {
+    const pluginFactory: PluginFactory = interopRequire(pluginName);
+    const loadedPlugin = pluginFactory(pluginOptions);
+
+    return {
+      ...loadedPlugin,
+      pluginName,
+    };
+  } catch (error) {
+    if (error.code === "MODULE_NOT_FOUND") {
+      options.logger.warn(
+        `🔍 Unable to load plugin ${pluginName}. Is he installed?`
```

Comment:
`Is it installed`

Reply:
> The hooks should IMHO receive meta data about the line like a key or a language it relates too, ideally all the meta we have. For example for the nbsp plugin the language is crucial :).

Passing info about the language into `transformLine` hook is implemented in https://github.com/AckeeCZ/lokse/pull/48